React/Views/RCTViewManager.m

```diff
@@ -259,6 +259,14 @@ - (RCTShadowView *)shadowView
     view.removeClippedSubviews = json ? [RCTConvert BOOL:json] : defaultView.removeClippedSubviews;
   }
 }
+RCT_CUSTOM_VIEW_PROPERTY(cornerSmoothing, BOOL, RCTView)
+{
+  if ([RCTConvert BOOL:json]) {
```

Comment:
`cornerCurve` seems to be available on iOS 13+ only. You can add `if (@available(iOS 13.0, *)) {` so it does nothing on older versions.